cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -32,18 +32,11 @@ class CircuitSerializer(serializer.Serializer):
     to the `serialize()` method of the class, which will produce a
     `Program` proto.  Likewise, the `deserialize` method will produce
     a `cirq.Circuit` object from a `Program` proto.
-
-    This class is more performant than the previous `SerializableGateSet`
-    at the cost of some extendability.
     """
 
-    def __init__(self, gate_set_name: str):
-        """Construct the circuit serializer object.
-
-        Args:
-            gate_set_name: The name used to identify the gate set.
-        """
-        super().__init__(gate_set_name)
+    def __init__(self):
+        """Construct the circuit serializer object."""
```

Comment:
Why do you prefer keeping it? I'm leaning towards removing it now because it's not currently useful, and it's easier to add a field later on than to remove one.

Reply:
Nm, ignore the last comment.  I thought you got rid of the doc string but not the argument.  This is fine.